person.rb

```diff
@@ -0,0 +1,36 @@
+require './nameable'
+
+class Person < Nameable
+  attr_reader :id, :rentals
+  attr_accessor :age, :name
+
+  def initialize(age, name = `unknown`, parent_permission: true)
+    super()
+    @id = Time.now.to_f
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rental = []
+  end
+
```

Comment:
- [ ] As stated above in Book class, kindly add `add_rental` method in the Person class because it has many reantals.